in_toto/models/link.py

```diff
@@ -106,6 +107,40 @@ def read(data):
     """Static method to instantiate a new Link from a Python dictionary """
     return Link(**data)
 
+  def display(self):
+    """
+    <Purpose>
+      Represents a Link by returning a string which contains information pertaining to its member fields
+    <Returns>
+      Link fields represented by strings:
+        type, name, 
+        inputs (materials), 
+        outputs (products)
+
+        Also returns "summary" string:
+          four detections: "created", "deleted", "modified" and "unchanged"
+    """
+    command = in_toto.util.check_string("{}".format(" ".join(self.command)))
+    materials = in_toto.util.check_string("{}".format(" ".join(self.materials)))
+    products = in_toto.util.check_string("{}".format(" ".join(self.products)))
+    summary = ""
+    new, deleted, modified, no_change = in_toto.util.compare_dictionaries(self.products, self.materials)
+
+    summary += "\n\tcreated: {}".format(in_toto.util.check_string(" ".join(new)))
+    summary += "\n\tdeleted: {}".format(in_toto.util.check_string(" ".join(deleted)))
+    summary += "\n\tmodified: {}".format(in_toto.util.check_string(" ".join(modified)))
+    summary += "\n\tunchanged: {}".format(in_toto.util.check_string(" ".join(no_change)))
```

Comment:
You could use one formatted string here instead, but I guess that's a matter of taste.

Reply:
Yup, doable. This change is reflected in the latest commit!